test/controllers/drivers_controller_test.rb

```diff
@@ -1,175 +1,226 @@
 require "test_helper"
 
 describe DriversController do
-  # Note: If any of these tests have names that conflict with either the requirements or your team's decisions, feel empowered to change the test names. For example, if a given test name says "responds with 404" but your team's decision is to respond with redirect, please change the test name.
+  let (:driver) {
+    Driver.create!(
+      name: "new driver", 
+      vin: "FJSKDJ12",
+      available: true
+    )
+  }
 
   describe "index" do
     it "responds with success when there are many drivers saved" do
-      # Arrange
-      # Ensure that there is at least one Driver saved
-
-      # Act
-
-      # Assert
-
+      get drivers_path
+      must_respond_with :success
     end
 
     it "responds with success when there are no drivers saved" do
-      # Arrange
-      # Ensure that there are zero drivers saved
-
-      # Act
-
-      # Assert
+      driver.destroy
```

Comment:
Just for the future there's a way to destroy all of a model.

```suggestion
      Driver.destroy_all
```

Fun when you just want to nuke the whole table.